.devcontainer/docker-compose.yml

```diff
@@ -19,6 +19,9 @@ services:
     init: true
     privileged: true
 
+    extra_hosts:
```

Comment:
This is required for the testcontainers to work on UNIX? Maybe would be great to introduce that into the template as well. It will not be automatically propagated but at least serves as example.

Also just wondering whether we still need the privileged set to true?

Reply:
Added to refactoring notes for later.
Not sure about the privileged part, though, need to doublecheck with what we have in other services

Reply:
I'd keep that as separate change for now, as this will probably need some investigation